assets/training/model_evaluation/environment/context/conda.yaml

```diff
@@ -36,7 +36,6 @@ dependencies:
   - pydantic=={{latest-pypi-version}}
   - mteb=={{latest-pypi-version}}
   - setuptools>=70.0.0
-  - rouge>=1.0.0,<2.0.0
```

Comment:
we are removing rouge as it is added as a dependency in latest version of azureml-metrics package.